### PR TITLE
feat:done quiz home page

### PR DESCRIPTION
**Pull request by tushargoyal812**
quiz-home

### CODE REVIEW

**Review by sanjay-xdr — APPROVED** [review]
everything looks good

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <div class="quiz-logo quiz-color">
+                <a class="quiz-color" href="">
+                    GarudaQuiz
+                </a>
+            </div>
+        </nav>
+    </header>
+    <main class="quiz-home-main flex centered">
+        <div class="quiz-img-wrapper mt-2">
+            <img src="/assets/undraw_transfer_money_rywa (1).svg" alt="" class="quiz-img">
```
Comment (syed-shaida-hussain) [inline index.html]: You can give alt text for image

File: mcq/hera-pheri/hera-pheri.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
+
+.option-btn{
+    width: 25rem;
+    padding: 1.5rem 8rem;
+    background-color: #cbd5e1;
```
Comment (syed-shaida-hussain) [inline mcq/hera-pheri/hera-pheri.css]: use variables for colors

**Review by syed-shaida-hussain — CHANGES_REQUESTED** [review]
Use variables for colors , everything else looks good.

File: index.html (hunk 1 above)
Comment (tushargoyal812) [reply index.html]: done

File: mcq/hera-pheri/hera-pheri.css (hunk 2 above)
Comment (tushargoyal812) [reply mcq/hera-pheri/hera-pheri.css]: done

**Pull request merged**